modules/openapi-generator/src/main/resources/dart2/api_client.mustache

```diff
@@ -209,12 +209,12 @@ class ApiClient {
     List<QueryParam> queryParams,
     Map<String, String> headerParams,
   ) {
-    authNames.forEach((authName) {
+    for(final authName in authNames) {
```

Comment:
Missing space

Reply:
where is the space missing? It's all good.

Reply:
`for (`. Whatever, we need to add post-processing to the generated examples so we have diffs that don't have this problem.